sdk/communication/azure-communication-identity/tests/test_communication_identity_client_async.py

```diff
@@ -49,7 +49,7 @@ async def test_create_user(self):
         assert user.properties.get('id') is not None
 
     @recorded_by_proxy_async
-    async def test_create_user_and_token(self):
+    async def test_create_user_and_token_chat(self):
```

Comment:
```suggestion
    async def test_create_user_and_token_with_scope_chat(self):
```
same comment for other tests for checking with diff scopes

Reply:
Please see my comment here https://github.com/Azure/azure-sdk-for-python/pull/28257#discussion_r1067106069

Reply:
In that comment you stating that async test with proxy don't support table tests (parametrized), but here I requested only renaming the test name for readable one, or am I missing smth?

Reply:
I might have misunderstood you. Anyway, it's a good suggestion, I've renamed